src/gcp_scanner/argparser.py

```diff
@@ -0,0 +1,96 @@
+"""This Module can be imported to parse command line arguments."""
+import argparse
+import logging
+
+def parse_arguments():
```

Comment:
Please follow the style of function description and arguments description that we use in all other functions.

Reply:
@mshudrak - ok sorry, I will add that, but can you please help me figure out how to remove the Readme.md change? I'm new to GitHub and not sure what happened or how to get that out of my branch - I checked the rebase stuff and thought I corrected it but guess not. Any help is much appreciated.

Reply:
Let's just address other comments first. I will help you with the README.md changes once you fix them.